### ISSUE TITLE
Fossa panics when analyzing (Newer Versions)

### ISSUE THREAD

**Issue by jose-bigio** [fossas/fossa-cli#141]
Had the following error when No go tools were installed. 

``` 
WARNING No supported Go build tools detected (try setting $DEP_BINARY or $GLIDE_BINARY or $GODEP_BINARY or $GOVENDOR_BINARY or $VNDR_BINARY): &errors.errorString{s:"could not resolve version"} &errors.errorString{s:"could not resolve version"} &errors.errorString{s:"could not resolve version"} &errors.errorString{s:"could not resolve version"} &errors.errorString{s:"could not resolve version"}
panic: runtime error: slice bounds out of range
```

This: panic: runtime error: slice bounds out of range 
does not seem good.

**Comment by corbin-coleman** [comment]
Lost the `WARNING` by installing `vndr`, but the panic stayed

```
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
github.com/fossas/fossa-cli/builders.getGoImports(0xc4200f02a0, 0xc420023360, 0x6, 0xb70379, 0x6, 0xc4200b66e8, 0x47, 0xc4200b66e0, 0x4f, 0x0, ...)
	/go/src/github.com/fossas/fossa-cli/builders/golang.go:485 +0x16d
github.com/fossas/fossa-cli/builders.(*GoBuilder).IsBuilt(0xc4200f02a0, 0xc420023360, 0x6, 0xb70379, 0x6, 0xc4200b66e8, 0x47, 0xc4200b66e0, 0x4f, 0x0, ...)
	/go/src/github.com/fossas/fossa-cli/builders/golang.go:743 +0x15e
main.doAnalyze(0xc420172e00, 0x10, 0x10, 0xc420023300, 0x6, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/fossas/fossa-cli/cmd/fossa/analyze.go:73 +0x360
main.analyzeCmd(0xc4200e6dc0)
	/go/src/github.com/fossas/fossa-cli/cmd/fossa/analyze.go:23 +0x109
github.com/fossas/fossa-cli/vendor/github.com/urfave/cli.HandleAction(0xa49d80, 0xb9e2a0, 0xc4200e6dc0, 0xc420072800, 0x0)
	/go/src/github.com/fossas/fossa-cli/vendor/github.com/urfave/cli/app.go:492 +0x7c
github.com/fossas/fossa-cli/vendor/github.com/urfave/cli.Command.Run(0xb711ad, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0xb8010c, 0x1a, 0x0, ...)
	/go/src/github.com/fossas/fossa-cli/vendor/github.com/urfave/cli/command.go:210 +0xa36
github.com/fossas/fossa-cli/vendor/github.com/urfave/cli.(*App).Run(0xc420095d40, 0xc42000c0a0, 0x2, 0x2, 0x0, 0x0)
	/go/src/github.com/fossas/fossa-cli/vendor/github.com/urfave/cli/app.go:255 +0x6a0
main.main()
	/go/src/github.com/fossas/fossa-cli/cmd/fossa/main.go:143 +0x1f21
```

**Comment by jose-bigio** [comment]
We experienced these errors off of the latest release version, and off of master.

**Comment by marcelormergulhao** [comment]
I had the same error here, even after installing go and dep.

The version used is the one suggested by the README:
fossa-cli version 0.6.6 (revision 607de55bec12623467f608ff49b895a373210c4b compiled with go version go1.10.3 linux/amd64).

Any tips on how to deal with it?

**Comment by jose-bigio** [comment]
@marcelormergulhao  (not speaking on behalf of FOSSA) personally the 0.5.1 release has been very stable. You can try using that, and see if it helps.

**Comment by marcelormergulhao** [comment]
@jose-bigio , thank you for the tip, it works perfectly so far!

**Comment by ilikebits** [comment]
This is due to bad tool detection logic in the v0.6.x series, which we're fixing in v0.7.0. The current v0.7.0 alpha release (v0.7.0-alpha.8, I believe) has support for Go.

**Comment by ilikebits** [comment]
This should be resolved in `v0.7.0-beta.1`. Please reopen if this is still an issue.
